### DISCUSSION TITLE
Sakız sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#258] [Soru Cevap]
Ahmet her gün elindeki sakızlardan 1 tanesini çiğneyip 3 yeni sakız alıyor. Başlangıçta Ahmet’in elinde 7 sakız varsa bunu dört katından fazlasına çıkarabilmesi için en az kaç gün geçmesi gerekiyor?

**Comment by alicinar11** [comment]
*3-1=2 buna göre:
*7+2=9
*9+2=11
*11+2=13
*13+2=15
*15+2=17
*17+2=19
*19+2=21
*21+2=23
*23+2=25
*25+2=27
*27+2=29>7×4
Cevap: 11 gün
